net-im/skypeforlinux/skypeforlinux-1.9.0.2.ebuild

```diff
@@ -10,7 +10,6 @@ inherit eutils rpm
 DESCRIPTION="P2P Internet Telephony (VoiceIP) client"
 HOMEPAGE="http://www.skype.com/"
 SRC_URI="https://repo.skype.com/rpm/stable/skypeforlinux_${PV}-1.x86_64.rpm"
```

Comment:
I suggest: `${P/-/_}` or `${PN}_${PV}` ;-). Also, you may eventually want to collapse the `-1` part into PV, in case they end up doing multiple builds with the same version.